### DISCUSSION TITLE
Should I be putting Flaresolverr on everything?

### DISCUSSION THREAD

**Discussion by devanteweary** [FlareSolverr/FlareSolverr#1116] [Q&A]
Just wondering if I should be assigning Flaresolverr to every indexer or if that will cause issues.
Right now I just have it on tgx and 1337x but not really sure if other ones need them or not.
I get connection fails every now and then from some of them, and then other times they seem to work fine.

So put flaresolverr on everything or nah?

Thank you!

**Comment by ilike2burnthing** [comment]
If you're referring to Prowlarr, it shouldn't make a difference to add it to all indexers, as FlareSolverr will only be used if there's a challenge.
